docs/docs/concepts/cloud-agent.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: cloud-agent
+title: Tracetest Cloud Agent (Beta)
+description: Tracetest Cloud Agent is a lightweight, dependency-free program that runs within the Tracetest Cloud Infrastructure.
```

Comment:
Change to:

description: Cloud Agent allows you to run Tracetest tests against public facing endpoints without deploying the Tracetest Agent.